app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  # paginates_per 5
```

Comment:
again commented code???
Make sure you review your PR before submitting

Reply:
done